packages/hub/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@textile/hub",
-  "version": "6.0.2",
+  "version": "6.0.3-alpha.0",
```

Comment:
all these just came from pushing an rc build changes to this. should autofix on next publish